cmd/atlas/atlas.go

```diff
@@ -37,15 +40,33 @@ var (
 func Execute() {
 	ctx := telemetry.NewContext()
 	rootCmd := atlas.Builder(&profile)
+	initSignalHandler(rootCmd)
 	if cmd, err := rootCmd.ExecuteContextC(ctx); err != nil {
-		telemetry.TrackCommand(telemetry.TrackOptions{
-			Cmd: cmd,
-			Err: err,
-		})
-		os.Exit(1)
+		handleError(cmd, err)
 	}
 }
 
+func initSignalHandler(cmd *cobra.Command) {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c,
+		syscall.SIGINT,  // CTRL-C
```

Comment:
according to [docs](https://cs.opensource.google/go/go/+/master:src/os/exec_posix.go;l=17) I think we should add os.Interrupt to the list, given in Windows it might be different

Reply:
os.Interrupt is the same as syscall.SIGINT, looking at the source code in exec_posix.go:

`	Interrupt Signal = syscall.SIGINT`